provides.py

```diff
@@ -125,3 +125,10 @@ def set_cohort_keys(self, cohort_keys):
         """
         for relation in self.relations:
             relation.to_publish['cohort-keys'] = cohort_keys
+
+    def set_default_cni(self, default_cni):
+        """
+        Send the default CNI.
```

Comment:
Mention what a typical `default_cni` value might be in this docstring.

Reply:
done